SPECS/security-tools.spec

```diff
@@ -0,0 +1,34 @@
+Name:           security-tools
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        Generates 3 UUIds for a pool secret
+License:        GPLv3
+URL:            https://github.com/xcp-ng/security-tools
+Source0:        https://github.com/xcp-ng/security-tools/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  cmake3
+BuildRequires:  make
+BuildRequires:  gcc
+
+%description
+Generates 3 UUIds for a pool secret.
+
+%prep
+%autosetup -p1
+
+%build
+mkdir build && cd build
+%cmake3 .. -DCMAKE_INSTALL_PREFIX=%{buildroot}/usr/bin
```

Comment:
`CMAKE_INSTALL_PREFIX` is just the installation prefix of the binary. The default value is `/usr/local`, you must set it to `%{buildroot}/usr` here.

See: https://github.com/xcp-ng/security-tools/blob/master/CMakeLists.txt#L72
`CMAKE_INSTALL_BINDIR` is the concatenation of `CMAKE_INSTALL_PREFIX` with `bin`.